components/conversations/CommentForm.js

```diff
@@ -104,6 +104,7 @@ const CommentForm = ({
   isDisabled,
   canUsePrivateNote,
   defaultType = commentTypes.COMMENT,
+  clickedComment,
```

Comment:
I think we can make this variable name more explicit:

```suggestion
  replyingToComment,
```